ui/raidboss/data/05-shb/trial/ruby-weapon-ex.js

```diff
@@ -2,10 +2,130 @@
 
 [{
   zoneRegex: {
-    en: /^Cinder Drift \(Extereme\)$/,
+    en: /^Cinder Drift \(Extreme\)$/,
   },
   timelineFile: 'ruby-weapon-ex.txt',
   triggers: [
+    {
+      id: 'RubyEx Optimized Ultima',
+      regex: Regexes.startsUsing({ source: 'The Ruby Weapon', id: '4ABE', capture: false }),
+      condition: function(data) {
+        return data.role == 'healer' || data.role == 'tank' || data.CanAddle();
+      },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'RubyEx Stamp',
+      regex: Regexes.startsUsing({ source: 'The Ruby Weapon', id: '4B03' }),
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'RubyEx Undermine',
```

Comment:
Sorry for being confused, but can you explain what's going on here? Why do you need the rubyCounter?

Reply:
Liquefaction is used when he does the dashes as well, so it's:

Undermine/Liquefaction
Liquefaction + Dashes
Undermine/Liquefaction
Liquefaction + Dashes

Reply:
OHHHH right.  Thanks!  Can you leave a comment about this?